README.md

```diff
@@ -6,14 +6,14 @@ Repository for out-of-tree scheduler plugins based on scheduler framework.
 
 Starting with release v0.18.9 container images are available in the official k8s container registry
 `k8s.gcr.io/scheduler-plugins/kube-scheduler`. See the [Compatibility Matrix section](#compatibility-matrix)
-for the list of images.
+for the list of images. Also, starting with scheduler-plugins release v0.19.8 multi-arch container images are
```

Comment:
A little advice is not necessary: can we replace this paragraph with the following chart? Just add a column
like this:
Scheduler Plugins  | Compiled With k8s Version | Container Image                                     |Arch         |
-------------------|---------------------------|-----------------------------------------------------|-----|
v0.19.8            | v1.19.8                   | k8s.gcr.io/scheduler-plugins/kube-scheduler:v0.19.8 |AMD64 ARM64 |
v0.18.9            | v1.18.9                   | k8s.gcr.io/scheduler-plugins/kube-scheduler:v0.18.9 |AMD64 |

Reply:
Fixed in commit fa0ae5f3e69206e49ecaae934affb304cef1a48f